mass_mailing_access/README.rst

```diff
@@ -0,0 +1,25 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+===========================
+Mass Mailing - Access Right
+===========================
+
+This module improve mass mailing module.
+Once installed, the module add new access right 'Mass Mailing / Manager'.
+Only manager can send mass mailing.
+Regular user will only have the possibility to create a mass mailing, and send a test email.
+
+Configuration
+=============
+
+Once installed, go to Settings / Users / Email Marketing and give 'Manager' Access to people allowed to send mass mailing.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Sylvain LE GAL <https://twitter.com/legalsylvain>
+* Druidoo <https://www.druidoo.io>
```

Comment:
can you split the README parts to separate files in the readme/ directory so that the README generation will work?